src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/ActionPlanEntityMapper.kt

```diff
@@ -13,7 +14,8 @@ import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.goal.ActionPl
 )
 interface ActionPlanEntityMapper {
   @ExcludeJpaManagedFields
-  fun fromDomainToEntity(actionPlan: ActionPlan): ActionPlanEntity
+  @GenerateNewReference
+  fun fromDomainDtoToEntity(createActionPlanDto: CreateActionPlanDto): ActionPlanEntity
```

Comment:
How about just `fromDtoToEntity`? DomainDto reads a bit weird to me and I thought it was a typo at first :)

Reply:
I could do, but the DTO classes live in the domain module
And I've have to rename it in loads of places!

I can do if you'd rather though ...

Reply:
Yeah it's ok. I might change it separately 😉 😂